.github/workflows/checks.yml

```diff
@@ -0,0 +1,37 @@
+name: Checks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  composer-normalize:
+    name: Composer Normalize
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v1
+
+      - name: Install
+        uses: docker://composer
+        with:
+          args: install
+
+      - name: Normalize
+        uses: docker://composer
+        with:
+          args: composer normalize --dry-run
+
+  roave-bc-check:
+    name: Roave BC Check
```

Comment:
Is there a way we can skip for master.. Should it run there?

Reply:
I would rather turn this off until we release a new version, but Tobias said tagging a new alpha would solve it.